tests/integration/templates/lambda_url.yaml

```diff
@@ -28,6 +29,10 @@ Resources:
           - Arn
       Handler: index.handler
       Runtime: python3.9
+      Environment:
+        Variables:
+          ENDPOINT_URL: aws.amazon.com
+
```

Comment:
I think we should do this via a different template file. It doesn't really make sense for the other test case(s) using the template.